composer.json

```diff
@@ -87,6 +87,7 @@
         "spryker-shop/cms-content-widget-product-connector": "^1.3.0",
         "spryker-shop/cms-content-widget-product-set-connector": "^1.3.1",
         "spryker-shop/date-time-configurator-page-example": "^0.3.1",
+        "spryker/acl-entity": "^1.9",
```

Comment:
ACL entity is currently used only in Merchant Portal, which is Marketplace-related functionality, it should not be part of `b2b-demoshop`, only of `b2b-mp` and `b2c-mp`.
Please remove the dependency, and get rid of all related changes. (Including new entities, and query classes and `<behavior name="\Spryker\Zed\AclEntity\Persistence\Propel\Behavior\AclEntityBehavior"/>`)